src/app/static/src/app/components/files/FileUpload.vue

```diff
@@ -58,13 +55,13 @@
         },
         data(): Data {
             return {
-                showUploadConfirmation: false
+                showUploadConfirmation: false,
+                uploading: true
```

Comment:
I think 'uploading' maybe isn't doing anything very much anymore? Disables the label and affects styling. Seems like it will always be true. I think this might be an artefact of this functionality moving into the ManageFile component?

Reply:
No, it ought to prevent them selecting another file, Nick is right, should be false by default and the :disabled flag should be on the input, not the label!

Reply:
This should now be fixed. Expected behaviour - after selecting a file to upload, "Select new file"  should become "..." and the file uploading temporarily disabled; after successful upload, the text reverts and you can upload a new file again.